tests/tests.rs

```diff
@@ -1911,3 +1911,27 @@ fn serde() {
         "invalid value: integer `-1`, expected u128",
     );
 }
+
+#[cfg(all(feature = "borsh", feature = "std"))]
```

Comment:
doesn't this also require "schema"?

Reply:
will add separate test for schemars. schemars is JSON Schema V7(OpenAPI schema). Used with serde (so requires serde).